python/lsst/ts/m2com/utility.py

```diff
@@ -123,6 +125,41 @@ def read_yaml_file(filepath: str | Path) -> dict:
     return content
 
 
+def read_error_code_file(filepath: str | Path) -> dict:
```

Comment:
Do you know what will be the valid types in the returned dictionary? Using `dict` implies `dict[Any, Any]`, but I imagine it is going to be something like `dict[str, str]`  or something like this no?

Reply:
Fixed in the commit: [9251138](https://github.com/lsst-ts/ts_m2com/pull/29/commits/925113864e2c5d488004305866687b3522d84d20). Thanks!